test_runner/regress/test_pageserver_secondary.py

```diff
@@ -895,6 +895,7 @@ def test_slow_secondary_downloads(neon_env_builder: NeonEnvBuilder, via_controll
     assert progress_3["bytes_total"] == progress_3["bytes_downloaded"]
 
 
+@pytest.mark.repeat(20)
```

Comment:
don't forget to remove before merge

Reply:
huh, I thought i pushed - obviously not

Reply:
https://github.com/neondatabase/neon/pull/10895/commits/9bd74239feeb3950c62c7cc77480153780ec06bb